hut_10sqft/test/test_suco.py

```diff
@@ -23,8 +23,7 @@
 from hut_10sqft.comp_mac_os import MacOsSetup
 from hut_10sqft.comp_ubuntu import UbuntuOsSetup
 from hut_10sqft.host_config import HostConf
-from hut_10sqft.suco_main import CompInitSetup
```

Comment:
For some reasons tests don't seem to be executed e.g. https://github.com/kinu-garage/hut_10sqft/actions/runs/19091903630/job/54544106298?pr=1300

Reply:
Tracked in https://github.com/kinu-garage/hut_10sqft/issues/1323